streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/metrics/flink/CheckPoints.java

```diff
@@ -82,5 +96,20 @@ public String getPath() {
   @Data
   public static class Latest implements Serializable {
     private CheckPoint completed;
+    private CheckPoint savepoint;
+
+    @JsonIgnore
+    public List<CheckPoint> getLatestCheckpoint() {
+      List<CheckPoint> checkPoints = new ArrayList<>();
+      if (Objects.nonNull(completed)) {
+        checkPoints.add(completed);
+      }
+      if (Objects.nonNull(savepoint)) {
+        checkPoints.add(savepoint);
+      }
+      return checkPoints.stream()
+          .sorted(Comparator.comparingLong(chk -> chk.id))
```

Comment:
The sort isn't necessary, right?

If so, we can return the checkpoints directly.